tests/testthat/test-dependencies.R

```diff
@@ -150,22 +150,24 @@ test_that("no warnings are produced when crawling dependencies", {
 
 })
 
-test_that("Suggests are dev. deps for non-package projects", {
+test_that("Suggests are dev. deps for all projects", {
+
   renv_tests_scope()
+
+  expected <- data.frame(
+    Package = "bread",
+    Dev = TRUE,
+    stringsAsFactors = FALSE
+  )
+
   writeLines(c("Type: Project", "Suggests: bread"), con = "DESCRIPTION")
   deps <- dependencies(dev = TRUE)
-  expect_true(nrow(deps) == 1)
-  expect_true(deps$Package == "bread")
-  expect_true(deps$Dev)
-})
+  expect_equal(deps[c("Package", "Dev")], expected)
```

Comment:
This construction is equivalent to the previous three lines but will give a more informative failure.